linux/deploy_vm/deploy_vm_from_iso.yml

```diff
@@ -134,10 +134,27 @@
         - unattend_install_conf is defined
         - unattend_install_conf is match('Photon') or unattend_install_conf is match('Ubuntu')
 
-    # Get guest IP
+    # VM IP might be changed at first boot. This is the first attempt to get VM IP and test it.
```

Comment:
Yes, it is a good idea. But it might be a big change, so I think it is better to improve update_inventory.yml in a separate PR. 
Or here makes some changes, which just wait for 5 min after autoinstall complete message is received, so that VM can have enough time to reboot and obtain an stable IPv4 address, then update inventory. Although it might don't need 5 minutes. Your opinion?